app/controllers/items_controller.rb

```diff
@@ -22,6 +22,28 @@ def show
     end
   end
   
+  def edit
+    @item = Item.find(params[:id])
+  end
+  
+  def update
+    @item = Item.find(params[:id])
+    if (current_user.items.include?(@item) and @item.update_attributes(item_params))
+      redirect_to item_path (@item), :notice => "#{@item.name} updated."
+    else
+      flash[:alert] = "#{@item.name} could not be updated: " + @item.errors.full_messages.join(",")
```

Comment:
Will the full message be really ugly or is it readable? "errors.full_messages"

Reply:
It will be ugly until we decide how we want to display error messages. This commit is pure functionality, but this is definitely a good point to bring up later